### PR TITLE
BRS-434 update correct park type on park details page

### PR DESCRIPTION
**Pull request by mayurmarakana89**
### Jira Ticket:
BRS-434

### Jira Ticket URL:
https://bcparksdigital.atlassian.net/browse/BRS-434

### Description:
Update correct park type on park details page

### CODE REVIEW

File: src/staging/src/components/park/about.js [hunk 1]
```diff
@@ -1,5 +1,9 @@
 import React from "react"
+<<<<<<< Updated upstream
 import { parseISO, format } from "date-fns"
+=======
+import { capitalize } from "lodash"
+>>>>>>> Stashed changes
```
Comment (cole) [inline src/staging/src/components/park/about.js:6]: Looks like a merge that didn't get resolved here

**Review by cole — CHANGES_REQUESTED** [review]
Looks good other than the merge error 🚀

File: src/staging/src/components/park/about.js (hunk 1 above)
Comment (mayurmarakana89) [reply src/staging/src/components/park/about.js]: Oops! Good catch! Have pushed the fix.

**Review by cole — APPROVED** [review]

**Review by marklise — COMMENTED** [review]
Not a huge fan of importing `_`.  There are simple ways to deal with both `sortBy` and `capitalize` in ES6.  Examples below:

```
const capitalize = ([firstLetter, ...restOfWord]) => firstLetter.toUpperCase() + restOfWord.join("");
```

```
const fruits = [
  {name:"banana", amount: 2},
  {name:"apple", amount: 4},
  {name:"pineapple", amount: 2},
  {name:"mango", amount: 1}
];

const sortBy = (key) => {
  return (a, b) => (a[key] > b[key]) ? 1 : ((b[key] > a[key]) ? -1 : 0);
};

fruits.concat().sort(sortBy("name"));

```

File: src/staging/src/utils/helpers.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+
+export const sortByKey = (key) => {
```
Comment (DayKirby-OXD) [inline src/staging/src/utils/helpers.js:3]: Nit: not seeing any reference to sortByKey. Could be removed, optional.

**Review by DayKirby-OXD — APPROVED** [review]
Looks good. At some point we might want ParkTypes in the CMS as a content type, but that seems like overkill at the moment, and definitely beyond scope of the ticket.

File: src/staging/src/utils/helpers.js (hunk 2 above)
Comment (mayurmarakana89) [reply src/staging/src/utils/helpers.js]: Good catch! Removed the unused helper function.

**Pull request merged**